From f56b72ed9c6bd0de2b7a64c5bb14be4707875504 Mon Sep 17 00:00:00 2001
From: Yves Vandervennet <r55763@freescale.com>
Date: Thu, 5 Feb 2009 21:11:17 -0600
Subject: [PATCH] ENGR00107230 Fix dryice build problem

   changed an #include from <> to "", as it allows
   build in tree.

Signed-off-by: Yves Vandervennet <r55763@freescale.com>
---
 drivers/mxc/security/dryice.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/mxc/security/dryice.c b/drivers/mxc/security/dryice.c
index 9483613..96c4035 100644
--- a/drivers/mxc/security/dryice.c
+++ b/drivers/mxc/security/dryice.c
@@ -25,7 +25,7 @@
 #define di_info(fmt, arg...) os_printk(KERN_INFO fmt, ##arg)
 #define di_warn(fmt, arg...) os_printk(KERN_WARNING fmt, ##arg)
 
-#include <sahara2/include/portable_os.h>
+#include "sahara2/include/portable_os.h"
 #include "dryice.h"
 #include "dryice-regs.h"
 
-- 
1.5.4.4

